### PR TITLE
Maja4 dev patch 1 Sample Geodata ToR

### PR DESCRIPTION
**Pull request by Maja4Dev**
These are the new Sample ToR for KfW Project location data collection and management which KfW counterparts need to amend and complete and then to forward to their project's Project Executing Agency (PEA) and / or their Implementation Consultant for data collection and updating.

### CODE REVIEW

**Review by fretchen — CHANGES_REQUESTED** [review]
I recommend a number of changes. 

- [ ] The changes on the index.md should not be in this branch as they are already present in  #20 . So we can wait until #20 is merged and then look for the diff or simply remove them from this branch.
- [ ] Geodat_ToR is not a markdown file yet. This should be changed. 
- [ ] This file should be added to the mkdocs.yml file line 15 or 16 such that it is also in the final compiled file...

As for the content I cannot judge....

**Comment by goergen95** [comment]
Hi @Maja4Dev,

thanks for opening the PR. There are three main points I am still uncomfortable with these ToR.

- The provision of the ToR are a burden to technical adapt partners. For partners with elaborated GIS systems there are better formats/procedures for the exchange of geospatial information. I would be against KfW making it mandatory for them to deliver their data in sub-optimal and proprietary data formats.
  
- Currently there is the lack of a clear process how the governance of future changes to the specification looks like. This is very much required if KfW wants to make it legally binding for partners to adhere to the specifiation. Then, partners need to know which version is the legally binding one, how changes are being handled on what that means for them during the lifetime of a project.
  
- KfW needs tooling to check if incoming data adheres to the standard. As long as KfW does not have such tooling it is really hard for anyone to assess the integrity of the data. Ideally, such tooling should be developed openly (e.g. in this repo) so that technical adapt partners can use the tools too to check their data before sending it to KfW.
  
Happy to further discuss this during a dedicated meeting.

**Comment by Maja4Dev** [comment]
Hi @goergen95 : The ToR are necessary for our KfW colleagues to be able to request the data collection from their implementing partners - regardless which data collection template will be used. 

Without ToR, no data will be collected for KfW. 

The filling out of the sample ToR (yellow-highlighted text) allows KfW staff to enter project-specific information which the data collector may need to have - such as the DAC-codes of the project, a.o.

Once we have better formats for data collection, we can easily add or replace them in the sample ToR. But there will be for many years still be some data collectors that will need the excel template.

Regarding the clear process how the governance of future changes to the specification looks like: from my perspective the (latest ) template published here in Github is the currently valid one. If we have more that one format (e.g. Excel and Kobo Toolbox and / or languages - they should all present an identical data model). Which other specifications do you need? Do you want to make a proposition?   
There should be definitely tooling to check if incoming data adheres to the standard. It would be great if this could be published here in Github, please discuss with @Jo-Schie .

**Comment by Maja4Dev** [comment]
Hi @fretchen,
happy to implement your recommendations, but need your help 😅.

**Comment by fretchen** [comment]
So I have a few propositions.

- The discussion on the process is moved into #23 
- The discussion on a streamlined data format can be found in #24
- I am not sure if the clear tooling needs an extra issue or if this is part of the other open PRs and issues @goergen95 ?

@Maja4Dev 
- I created a PR to implement most of the suggested changes [here](https://github.com/Maja4Dev/open-geodata-model/pull/2).
- However, I also think that we still need to sit together to fix some layout problems...

**Review by fretchen — APPROVED** [review]
LGTM now especially as the content of the already existing RMMV Guidebook did not change.

File: docs/sample_tor.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+-	All planned or existing project locations and their respective attributes shall be inserted into the template row by row. One row is equivalent to one project location. In case it is not self-evident how the locations of the project are to be defined (e.g. in some cases, it makes sense to summarize a number of small projects in one community under one project location, in other cases it doesn’t), this shall be clarified with the recipient parties of the project reports. 
+-	The DAC 5 Purpose Classification(s), also called CRS-Code(s) of the project are the following: {==please insert all (five-digit) CRS-Codes assigned to your project in INPRO – these are one up to four codes==}. 
+Each location is to be assigned at least one CRS-code (mandatory). If one location has more than one code assigned to, create an additional row for each additional code containing the same GPS-coordinates. For details, please check [the guidelines](index.md).
+-	The Following Data Fields stated as Non-Mandatory in the Excel/Kobo-Template are to be filled out for this project {== (boxes below to be checked by KfW) ==}:
```
Comment (fretchen) [inline docs/sample_tor.md:25]:
```suggestion
-	The Following Data Fields stated as Non-Mandatory in the Excel Template are to be filled out for this project {== (boxes below to be checked by KfW) ==}: 
```

File: docs/sample_tor.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    * [ ] Related Community / Village / Neighborhood
+
+-	For each project location there should be one **exact or approximate geolocation** dataset, which is to be updated in the above-stated regular reporting intervals. The Details are explained in the [guildelines](index.md)
+-	The frequency of location data reporting and changes is in accordance to the general KfW reporting requirements (chapter 6.1: Progress of Services and Works) –annually or bi-annually or other, depending on the type of project (frequently changing location data or not, see also chapter 1). 
+-	If a {== Geographic Information System (GIS), (Remote) Management Information System (R/MIS) or Maintenance Management System (MMS) (*please choose which*) ==} is used in the project, then the existing location IDs can be entered also in the Excel / Kobo template as Project-Specific Location Identifiers (Column G in the Table “Fill-Me” in the project location data [Excel template](https://github.com/openkfw/open-geodata-model/blob/main/Project_Location_Data_Template_V02.xlsx). This makes it easier to check location data across project partner systems and verify the use of funds.
```
Comment (fretchen) [inline docs/sample_tor.md:38]:
```suggestion
-	If a {== Geographic Information System (GIS), (Remote) Management Information System (R/MIS) or Maintenance Management System (MMS) (*please choose which*) ==} is used in the project, then the existing location IDs can be entered also in the Excel template as Project-Specific Location Identifiers (Column G in the Table “Fill-Me” in the project location data [Excel template](https://github.com/openkfw/open-geodata-model/blob/main/Project_Location_Data_Template_V02.xlsx). This makes it easier to check location data across project partner systems and verify the use of funds.
```

File: docs/sample_tor.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+## 4.	Data security and privacy
+
+-	Europe-based Consultants or Project Executing Agencies / Implementing Partners or Consultants working directly for KfW are legally required to ensure compliance of data collection incl. websites and software both with the relevant national regulations of the country in which the data is being collected as well as with the General Data Protection Regulation (GDPR, in Germany called DSGVO) all other data providers are required to adhere to the relevant national regulations of the country in which the data is being collected.
+-	Data shall only be transferred through encrypted channels (e.g. KfW data room). 
+-	Special caution is required in fragile and conflict contexts, where location data should be only treated with extra diligence e.g. working only with approximate geo coordinates for security reasons. 
+
+The [KfW’s Project Location Geodata Model](https://github.com/openkfw/open-geodata-model) is part of these ToR including the Excel / Kobo Template as well as the Location Type Icons currently being updated and both to be found under the same link above.
```
Comment (fretchen) [inline docs/sample_tor.md:52]:
```suggestion
The [KfW’s Project Location Geodata Model](https://github.com/openkfw/open-geodata-model) is part of these ToR including the Excel Template as well as the Location Type Icons currently being updated and both to be found under the same link above. 
```

**Pull request merged**